content/en/blog/_posts/2024-05-01-secret-sync-controller-alpha.md

```diff
@@ -0,0 +1,62 @@
+
+2. Standard Kubernetes Secret Access: Secrets can be accessed via volume mounting, environment variables, or via the Kubernetes API. Workloads and ingress controllers do not need to be customized to access a cloud provider secret store and developers have options on how to access secrets.  
+
+3. Security: The Secret Sync Controller (SSC) has limited permissions and leverages the latest Kubernetes security features (complete list below) so that cluster admins do not need to configure and restrict permissions themselves.  
+
+## How to use the Secret Sync Controller
+
+The Secret Sync Controller (SSC) must be used alongside a [secret store Cloud Provider](https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html#install-external-secret-providers) that implements the Secret Store CSI Driver provider interface to provide an end-to-end solution for synchronizing secrets from the cloud secret store to the Kubernetes secret store.  
+
+1. Install the [secret sync controller](https://gist.github.com/aramase/46bd3d4270d9c44b59e8c2afe56fbc09) via a Helm chart, or YAML manifests alongside a secret store provider. If using the helm chart, users can configure parameters such as sync intervals and resource limitations at install time.
+
+2. Configure an identity with permission to read the cloud provider's secrets.  
+
+3. Configure the cloud provider with a secret provider class CustomResourceDefinition (CRD).  
+
+4. Configure the Secret Sync Controller with a secret sync CRD that contains the associated cloud provider CRD, name of the secret in the cluster, target namespace, and the Kubernetes service account that has federated identity access to the Cloud Provider secret.  
+
+5. Apply the CRDs in the cluster and secrets will automatically begin syncing.  
+
+## Security features
+
+By default, the Secret Sync Controller (SSC) enables the following security features:
+
+1. Isolation of Functionality: The SSC splits out the functionality of syncing secrets from a cloud provider (via a secret sync Provider) and storing the secrets in a cluster (the SSC). Each component can have much more limited permissions, minimizing the risk of a cluster compromise.  
+
+2. Limited Permissions via ValidatingAdmissionPolicies: The SSC is a privileged facility that should be in a protected namespace, such as `kube-system`. The SSC leverages ValidatingAdmissionPolicies to limit its capabilities, ensuring it cannot perform operations such as deleting Secrets, or editing Secrets beyond its designated scope (defined via labels and annotations). Validating admission policies will be generally available and enabled by default starting with Kubernetes 1.30; right now, they are beta and opt-in.
```

Comment:
```suggestion
2. Limited Permissions via ValidatingAdmissionPolicies: The SSC is a privileged facility that should be in a protected namespace, such as `kube-system`. The SSC leverages ValidatingAdmissionPolicies to limit its capabilities, ensuring it cannot perform operations such as deleting Secrets or editing Secrets beyond its designated scope (defined via labels and annotations). Validating admission policies will be generally available and enabled by default starting with Kubernetes 1.30; right now, they are beta and opt-in.
```

If these features are strictly sequential or i'm planning to reference them later on easily, i would keep the number as it is. Otherwise I would use bullets to list the security features for visual simplicity.